core/domain/services/assets/CoreAssetManager.php

```diff
@@ -585,17 +591,21 @@ private function registerAdminAssets()
     {
         $this->addJavascript(
             CoreAssetManager::JS_HANDLE_WP_PLUGINS_PAGE,
-            $this->registry->getJsUrl($this->domain->assetNamespace(), 'wp-plugins-page'),
-            array(
-                CoreAssetManager::JS_HANDLE_JQUERY,
-                CoreAssetManager::JS_HANDLE_VENDOR,
+            $this->registry->getJsUrl($this->domain->assetNamespace(), CoreAssetManager::JS_HANDLE_WP_PLUGINS_PAGE),
+            $this->registry->getJsDependencies(
+                $this->domain->assetNamespace(),
+                CoreAssetManager::JS_HANDLE_WP_PLUGINS_PAGE
             )
         )
         ->setRequiresTranslation();
 
         $this->addStylesheet(
             CoreAssetManager::JS_HANDLE_WP_PLUGINS_PAGE,
-            $this->registry->getCssUrl($this->domain->assetNamespace(), 'wp-plugins-page')
+            $this->registry->getCssUrl($this->domain->assetNamespace(), CoreAssetManager::JS_HANDLE_WP_PLUGINS_PAGE),
+            $this->registry->getCssDependencies(
+                $this->domain->assetNamespace(),
+                CoreAssetManager::JS_HANDLE_WP_PLUGINS_PAGE
```

Comment:
looks like this was an existing issue, but we are using a JS handle for a stylesheet asset

Reply:
ya I thought the same thing.  It's this way because the handles are the same.  I think we'll leave for now until/when the handles vary.  Might be good for cases like this to just have a more generic `ASSET_HANDLE_WP_PLUGINS_PAGE` const instead for cases where there's no difference between the css and js handle.  We can do in a separate pull.